bundles/persistence/org.openhab.persistence.jdbc/java/org/openhab/persistence/jdbc/db/JdbcBaseDAO.java

```diff
@@ -77,6 +87,42 @@ public JdbcBaseDAO() {
         initSqlQueries();
     }
 
+    /**
+     * ## Get high precision by fractal seconds, examples ##
+     *
+     * mysql > 5.5 + mariadb > 5.2:
+     * DROP TABLE FractionalSeconds;
+     * CREATE TABLE FractionalSeconds (time TIMESTAMP(3), value TIMESTAMP(3));
+     * INSERT INTO FractionalSeconds (time, value) VALUES( NOW(3), '1999-01-09 20:11:11.126' );
```

Comment:
To be more specific, if the documentation is correct and someone is using MySQL 5.6.2 or 5.6.3, TIMESTAMP(n) and NOW(n) will ignore n, but I suppose that's not too bad.  Maybe a small note in the JDBC wiki page on the subject is warranted?